service/src/test/java/com/commercetools/pspadapter/payone/domain/payone/model/common/AuthorizationRequestTest.java

```diff
@@ -0,0 +1,270 @@
+        request.setTitle(title);
+        request.setUserid(userId);
+        request.setVatid(vatId);
+        request.setZip(zip);
+    }
+
+    @After
+    public void tearDown() {
+        System.clearProperty(HIDE_CUSTOMER_PERSONAL_DATA);
+    }
+
+    @Test
+    public void createsFullMap() {
+        assertThat(request.toStringMap(false)).containsOnly(
+                entry("narrative_text", narrativeText),
+                entry("param", param),
+                entry("key", keyHash),
+                entry("clearingtype", clearingType),
+                entry("errorurl", errorUrl),
+                entry("vatid", vatId),
+                entry("request", requestType),
+                entry("backurl", backUrl),
+                entry("successurl", successUrl),
+                entry("mode", mode),
+                entry("amount", amount),
+                entry("reference", reference),
+                entry("currency", currency),
+                entry("api_version", apiVersion),
+                entry("portalid", portalId),
+                entry("mid", merchantId),
+                entry("userid", userId),
+                entry("customerid", customerId),
+                entry("personalid", personalId),
+                entry("language", language),
+                entry("birthday", birthday),
+                entry("gender", gender),
+                entry("salutation", salutation),
+                entry("title", title),
+                entry("lastname", lastName),
+                entry("firstname", firstName),
+                entry("country", country),
+                entry("state", state),
+                entry("city", city),
+                entry("street", street),
+                entry("company", company),
+                entry("addressaddition", addressAddition),
+                entry("zip", zip),
+                entry("telephonenumber", telephoneNumber),
+                entry("ip", ip),
+                entry("shipping_firstname", shippingFirstName),
+                entry("shipping_lastname", shippingLastName),
+                entry("shipping_country", shippingCountry),
+                entry("shipping_state", shippingState),
+                entry("shipping_city", shippingCity),
+                entry("shipping_street", shippingStreet),
+                entry("shipping_company", shippingCompany),
+                entry("shipping_zip", shippingZip),
+                entry("email", email));
+
+    }
+
+    @Test
+    public void createsMapWithHiddenSecrets() {
+        assertThat(request.toStringMap(true)).containsOnly(
+                entry("narrative_text", ClearSecuredValuesSerializer.PLACEHOLDER),
+                entry("param", ClearSecuredValuesSerializer.PLACEHOLDER),
+                entry("key", ClearSecuredValuesSerializer.PLACEHOLDER),
+                entry("clearingtype", clearingType),
+                entry("errorurl", errorUrl),
+                entry("vatid", vatId),
```

Comment:
What about this VAT ID? Isn't it also personal info?

Reply:
@floriansattler ?

Reply:
It`s a public info, as it is from companies.